static/docs/commands-reference/import-url.md

```diff
@@ -215,8 +222,9 @@ file has changed.
 ## Example: Detecting remote file changes
 
 What if that remote file is one which will be updated regularly? The project
-goal might include regenerating some artifact based on the updated data. A
-pipeline can be triggered to re-execute based on a changed external dependency.
+goal might include regenerating a <abbr>data artifact</abbr> based on the
+updated source. A pipeline can be triggered to re-execute based on a changed
```

Comment:
we are changing this semantics ... we can update the docs and examples a bit later when we do docs for the `dvc update`

Reply:
I'm not sure what you mean. Can you clarify?

p.s. I didn't change that sentence, it's just a rewrap due to the `<abbr>`

Reply:
yes, sure. So, by default all external dependencies are going to be locked (`lock: true` and `-rev-lock: <git-commit-hash>` in the DVC-file). It means when you run `dvc repro` or `dvc status` they are not going to check the changes automatically. You will need to explicitly call `dvc update` to fetch the new data, update the `-rev-lock` field

Reply:
OK great added note in https://github.com/iterative/dvc.org/issues/478#issuecomment-510681023.